### PR TITLE
feat(kustomize): adding deployment via kustomize

### PR DESCRIPTION
**Pull request by pawanpraka1**
fixes : https://github.com/openebs/zfs-localpv/issues/214

Signed-off-by: Pawan <pawan@mayadata.io>

### CODE REVIEW

**Comment by codegagan** [comment]
Should not this be added in ci to test deployment via (-k) and then run all/some sanity tests ?

**Review by spencergilbert — APPROVED** [review]
Looks straight forward :+1:

**Comment by pawanpraka1** [comment]
Thanks @spencergilbert .

>Should not this be added in ci to test deployment via (-k) and then run all/some sanity tests ?

@codegagan will you be able to work on that part?

File: deploy/yamls/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+images:
+  - name: quay.io/openebs/zfs-driver
+    newTag: ci
+resources:
+- namespace.yaml
+- zfsvolume-crd.yaml
+- zfssnapshot-crd.yaml
+- zfsbackup-crd.yaml
+- zfsrestore-crd.yaml
+- zfs-driver.yaml
```
Comment (prateekpandey14) [inline deploy/yamls/kustomization.yaml:10]: Do we need to update README with the command to use the kustomize based install using `-k` with `kubectl apply` ?

**Review by prateekpandey14 — APPROVED** [review]
looks good, a small comment for adding install cmd

File: deploy/yamls/kustomization.yaml (hunk 1 above)
Comment (pawanpraka1) [reply deploy/yamls/kustomization.yaml]: added.

**Review by kmova — APPROVED** [review]

**Pull request merged**